### DISCUSSION TITLE
Question : mach speed and wind_speed

### DISCUSSION THREAD

**Discussion by bdance7** [Estia-advanced-programming/pandora-public#81] [Q&A] [answered]
hello, we're lost to caculate those two variables.
should we calculate the speed with the time delta between each report or air_speed = the speed of the plane?
similar question for the wind speed, we can't see the variable that can be useful to us ...
thank you in advance

**Comment by dhmmasson** [comment, marked as answer]
related to #28 
Air speed is the speed reported by the instrument on board ( as measured by an anemometer ) 
ground speed is the speed you compute from the gps coordinate 
wind speed is just the difference
